kernel/aster-nix/src/syscall/arch/mod.rs

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: MPL-2.0
+
+#[cfg(target_arch = "x86_64")]
+pub mod x86;
+#[cfg(target_arch = "x86_64")]
+pub use self::x86::*;
```

Comment:
Add a module-level Rust doc to explain what are exposed by this module, which is unclear as they are generated by macros and re-exported with `x86::*`.

```
//! Implement the `syscall_dispatch` function and the const values of system call number such as `SYS_READ`.
```